protocol/migrations/sqlite/1670921937_add_album_id.up.sql

```diff
@@ -1 +0,0 @@
-ALTER TABLE user_messages ADD COLUMN album_id VARCHAR;
```

Comment:
I don't think we should remove any existing migrations. Instead we need a migration that moves album_id s over to the new column after it has been created

Reply:
Yes, this will break the app on some devices, migrations should never be removed